servers/quarkus-server/src/main/resources/application.properties

```diff
@@ -172,6 +175,8 @@ quarkus.container-image.name=nessie
 %test.quarkus.security.users.embedded.roles.admin_user=admin,user
 %test.quarkus.security.users.embedded.roles.test_user=test123
 %test.nessie.version.store.jgit.type=INMEMORY
+%test.quarkus.keycloak.devservices.enabled=false
```

Comment:
Do we need these two properties here or could those go into the test-profile?

Reply:
`devservices` configs must be set at build time.... but let me try if setting them in test profiles work.

Reply:
Even though Quarkus docs say it defaults to `true`, in practice it turned out setting this parameter was not necessary.... removed in 089e6c36d10a943566eff5e094a161b4653da443